docs/topics/feed-exports.rst

```diff
@@ -295,7 +295,7 @@ If you enable batching, you must also :ref:`set feed URIs appropriately with pla
 Default Batch Handler
 ----------------------
 
-.. autoclass:: scrapy.extensions.batches.BatchHandler
+.. autoclass:: scrapy.extensions.batches.DefaultBatchHandler
```

Comment:
I think it may be better to leave the previous name. The new name could be quite confusing if we switch the default value to a different one in the future.